graylog2-server/src/main/java/org/graylog/plugins/views/search/validation/LuceneQueryParser.java

```diff
@@ -34,13 +34,14 @@
 
     public LuceneQueryParser() {
         this.parser = new TermCollectingQueryParser(ParsedTerm.DEFAULT_FIELD, new StandardAnalyzer());
+        this.parser.setSplitOnWhitespace(true);
```

Comment:
@janheise This is the most important change, otherwise the `and and and or or or` parsing failed as you noticed. Thx!